### PR TITLE
Spaceland - BACK TO THE ASTEROID

### PR DESCRIPTION
**Pull request by ToasterBiome**
# Document the changes in your pull request

image

asteroid. reminiscent of the old style, with a new mining base in the south.

WE ARE HERE TO MINE, MINE WE WILL

no special bosses (yet)
no ruins (yet)
requires spaceland minetype (might just make it randomly choose)

# Wiki Documentation

Spaceland added to yogstation

# Changelog

:cl:  
mapping: Adds spaceland (the old asteroid)
/:cl:

### CODE REVIEW

**Comment by RG4ORDR** [comment]
YES.
APPROVE.
MERGE.
NOW.

**Comment by neeshacark** [comment]
YES.
APPROVE.
MERGE.
NOW.

**Comment by azzzertyy** [comment]
keep it without bosses.

**Comment by Aquizit** [comment]
> keep it without bosses.

"Guy McMiner has signed up as Shaft Miner."
Miner learns it's asteroid.
"Guy McMiner has been moved to cryogenic storage."

**Comment by MajManatee** [comment]
pogging rn just a little.
Interested to see what kind of bosses you come up with. In the meantime why not just reuse the lava land ones? Ruins too.

**Comment by Addust** [comment]
_happy imperialism noises_
image

**Comment by TerraTheUnwise** [comment]
wait I just noticed why is the UK there.

also is this a ghost spawn or an alternate place that the mining shuttle can go?

**Comment by azzzertyy** [comment]
> > keep it without bosses.
> 
> "Guy McMiner has signed up as Shaft Miner." Miner learns it's asteroid. "Guy McMiner has been moved to cryogenic storage."

dont see the problem here :)

**Comment by UltraBigRat** [comment]
> > > keep it without bosses.
> > 
> > 
> > "Guy McMiner has signed up as Shaft Miner." Miner learns it's asteroid. "Guy McMiner has been moved to cryogenic storage."
> 
> dont see the problem here :)

No mats for you today

**Comment by N3D6** [comment]
I love this but I have one problems

Slowdown will be an issue because mining with a slowdown is actual cock and nuts torture

I can make ruins if you want :3333

**Comment by Moltijoe** [comment]
> > keep it without bosses.
> 
> "Guy McMiner has signed up as Shaft Miner." Miner learns it's asteroid. "Guy McMiner has been moved to cryogenic storage."

Dont call out john miner like that

**Comment by ynot01** [comment]
still wip?

File: code/controllers/subsystem/mapping.dm [hunk 1]
```diff
@@ -317,13 +317,21 @@ SUBSYSTEM_DEF(mapping)
...
+	if(config.minetype == "random")
+		minetype = pick("lavaland","spaceland")
+	switch(minetype)
+		if("lavaland")
+			LoadGroup(FailedZs, "Lavaland", "map_files/mining", "Lavaland.dmm", default_traits = ZTRAITS_LAVALAND) //Yogs, yoglavaland
+		if("icemoon")
+			LoadGroup(FailedZs, "Ice moon", "map_files/mining", "Icemoon.dmm", default_traits = ZTRAITS_ICEMOON)
+			LoadGroup(FailedZs, "Ice moon Underground", "map_files/mining", "IcemoonUnderground.dmm", default_traits = ZTRAITS_ICEMOON_UNDERGROUND)
+		if("spaceland")
+			LoadGroup(FailedZs, "Spaceland", "map_files/mining", "Spaceland.dmm", default_traits = ZTRAITS_SPACE)
```
Comment (ynot01) [inline code/controllers/subsystem/mapping.dm:330]: `ZTRAIT_SPACE` lacks `ZTRAIT_MINING`

**Review by ynot01 — CHANGES_REQUESTED** [review]

**Review by Bibby0110 — CHANGES_REQUESTED** [review]
Make it not spawn randomly or finish it

**Comment by ToasterBiome** [comment]
later

**Pull request closed without merging**